src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -561,6 +567,13 @@ impl A320Hydraulic {
             &self.aft_cargo_door,
             self.yellow_loop.pressure(),
         );
+
+        self.slats_flaps_complex.update(
+            context,
+            self.green_loop.pressure(),
+            self.blue_loop.pressure(),
+            self.yellow_loop.pressure(),
```

Comment:
```suggestion
            &self.green_loop,
            &self.blue_loop,
            &self.yellow_loop,
```

I am not sure if you're planning to implement this with the new hydraulics refactor PR, but in my eyes this would be one of those times where it's a good idea to pass in the hydraulic loops directly and have the `slats_flaps_complex.update` method accept a trait exposing a `pressure()` function. This is described in the guidelines under "Passing data around".
```rust
trait HydraulicLoop {
    fn pressure(&self) -> Pressure;
}
```

Reply:
Yes this is planned in big hydraulic PR so I just made it quick and simple. The whole flapgear system added here will be replaced by hydraulic one as soon as this PR gets merged too.